server/events/command_runner_test.go

```diff
@@ -522,6 +522,23 @@ func TestRunCommentCommand_UnmatchedBranch(t *testing.T) {
 	vcsClient.VerifyWasCalled(Never()).CreateComment(matchers.AnyModelsRepo(), AnyInt(), AnyString(), AnyString())
 }
 
+func TestRunCommentCommand_ClosedPull_Unlocck(t *testing.T) {
+	t.Log("if an unlock command is run on a closed pull request, atlantis should" +
+		" invoke the delete command and comment on PR accordingly")
+	vcsClient := setup(t)
+	pull := &github.PullRequest{
+		State: github.String("closed"),
+	}
+	modelPull := models.PullRequest{BaseRepo: fixtures.GithubRepo, State: models.ClosedPullState, Num: fixtures.Pull.Num}
+	When(githubGetter.GetPullRequest(fixtures.GithubRepo, fixtures.Pull.Num)).ThenReturn(pull, nil)
+	When(eventParsing.ParseGithubPull(pull)).ThenReturn(modelPull, modelPull.BaseRepo, fixtures.GithubRepo, nil)
+
+	ch.RunCommentCommand(fixtures.GithubRepo, &fixtures.GithubRepo, nil, fixtures.User, fixtures.Pull.Num, nil)
```

Comment:
This is incorrect, you need to pass in a command context for the last parameter specifying `&events.CommentCommand{Name: command.Unlock}`.

Ex: https://github.com/runatlantis/atlantis/blob/480f70b16905b8b5567a202b4237f500c6246786/server/events/command_runner_test.go#L554

Reply:
Fixed.